src/TokenFuzzChecks.sol

```diff
@@ -0,0 +1,456 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity >=0.8.0;
+
+import "./TokenChecks.sol";
+
+/** 
+ * @title Token Fuzz Checks
+ * @dev Contains fuzz checks for token contracts similar to test/SampleToken.sol
+ */
+contract TokenFuzzChecks is TokenChecks {
+
+    using GodMode for *;
+
+    // ************************************************************************************************************
+    // Mint/Burn
+    // ************************************************************************************************************
+
+    function fuzzCheckMintBurn(
```

Comment:
Ah sorry, it's here, just got confused since `fuzzCheckBurnInsufficientBalance` and `fuzzCheckTokenModifiers` are missing.
We should probably add those here and see other are not missing.